ciao-cli/image.go

```diff
@@ -65,6 +71,8 @@ func (cmd *imageAddCommand) parseArgs(args []string) []string {
 	cmd.Flag.StringVar(&cmd.id, "id", "", "Image UUID")
 	cmd.Flag.StringVar(&cmd.file, "file", "", "Image file to upload")
 	cmd.Flag.StringVar(&cmd.template, "f", "", "Template used to format output")
+	cmd.Flag.BoolVar(&cmd.public, "public", false, "Public Image")
```

Comment:
@markdryan, agree, I just wanted to use tags for putting internal images information. But agree, let's allow public and private images have tags.
Regarding visibility flag, I like it. /me changing it